lib/ros_value.h

```diff
@@ -320,6 +323,29 @@ class RosValue {
     return values;
   }
 
+  // Used for python bindings
```

Comment:
I think very few people are going to be familiar with what pybind11::buffer_info does - 
Could you write a better comment explaining what exactly this does and how one might properly use it?

Reply:
Sure thing!

Reply:
Done